torchtext/data/utils.py

```diff
@@ -155,6 +155,15 @@ def get_tokenizer(tokenizer, language='en'):
         except ImportError:
             print("Please install revtok.")
             raise
+    elif tokenizer == "sentencepiece" and spm_name is not None:
```

Comment:
This tokenizer is special in that there's a file it uses to be created. That makes it different from all the other ones. Maybe it should't be here?